src/main/java/com/nexters/momo/member/auth/utils/AuthorizationExtractor.java

```diff
@@ -0,0 +1,36 @@
+package com.nexters.momo.member.auth.utils;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.apache.logging.log4j.util.Strings;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Enumeration;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class AuthorizationExtractor {
+
+    public static final String AUTHORIZATION = "authorization";
+    public static final String ACCESS_TOKEN_TYPE = AuthorizationExtractor.class.getSimpleName() + ".ACCESS_TOKEN_TYPE";
+
+    public static String extract(HttpServletRequest request, AuthorizationType type) {
+        String typeToLowerCase = type.toLowerCase();
+        int typeLength = typeToLowerCase.length();
+
+        Enumeration<String> headers = request.getHeaders(AUTHORIZATION);
```

Comment:
```suggestion
        Enumeration<String> headers = request.getHeaders(HttpHeaders.AUTHORIZATION);
```
새롭게 만들어서 쓰기보다 이미 내부에 구현되어 있는 상수들을 이용하면 좋을 것 같아 제안 드립니닷ㅎㅎ

Reply:
엇 ㅋㅋㅋ 그러네요 변경하겠습니닷!! 감사합니닷!